### PR TITLE
Various Fixes

### LINKED ISSUE #2509: Redirect to stack URL when note is moved into stack
Currently when in StackView and the viewed note gets moved into a stack (while not being the parent note), the URL does not change and the StackView is showed as if the viewed note is the parent note (aka. the note is not stacked).

I think it would be intuitive to redirect to the StackView URL so that the current state of the note is mirrored (as in "the note becomes a child note in a stack")

### PR DESCRIPTION
**Pull request by Resaki1**
## Description

Multiple things have been fixed: Proper redirect when note that is currently viewed in StackView is stacked or deleted by another user, removed deprecated tooltip attribute and un-reversed reversed component sequence. Closes #2509 

## Changelog

- When a user has a note open in stack view, and another user moves that note to another stack, the first user is redirected to the new stack and informed about it via toast. On Deletion, the user is redirected to the board.
- The striped background of hidden columns now uses the accent color of the column, so that it is easier to differentiate between adjacent hidden columns
- Some instances of the tooltip component used deprecated attributes. This has been fixed
- The order of the column header and noteinput where reversed in jsx and using css `flex-direction: column;` un-reversed. This has been fixed as well
- some comments to better explain the different useEffects in StackView

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] The light- and dark-theme are both supported and tested
- [x] The design was implemented and is responsive for all devices and screen sizes
- [x] The application was tested in the most commonly used browsers (e.g. Chrome, Firefox, Safari)

### CODE REVIEW

File: public/locales/en/translation.json [hunk 1]
```diff
@@ -223,7 +223,9 @@
     "moderatorResetReadyStates": "Do you want to reset the readiness of the participants?",
     "moderatorResetReadyStatesButton": "Reset!",
     "noteToHiddenColumn": "You just added a note to a hidden column.",
-    "noteToHiddenColumnButton": "Reveal column"
+    "noteToHiddenColumnButton": "Reveal column",
+    "noteMovedToStack": "The note was moved to a stack.",
+    "noteDeleted": "The note was deleted."
```
Comment (brandstetterm) [inline public/locales/en/translation.json:228]: As far as I know, it's a best practice to not end sentences in toasts with a period. I also think that's quite inconsistent in our translation files 😆

File: src/components/Board/Board.scss [hunk 2]
```diff
@@ -2,18 +2,16 @@
 @import "components/Column/Column";
 
 .board {
-  --n: 1;
+  --visible-columns: 1;
```
Comment (brandstetterm) [inline src/components/Board/Board.scss:5]: Thank you for improving the naming ❤️

**Comment by brandstetterm** [comment]
I've noticed that the striped background using the accent color of hidden columns is quite hard to see for some colors (e.g. poker purple, lean lilac, planning pink). At first sight, I thought it was a bug that for some columns the background isn't stripped.

Screenshot 2023-03-08 at 09 34 44


Also, since you've removed the spacer on the left and right sides, columns are now of different widths. 
Screenshot 2023-03-08 at 09 36 36

Screenshot 2023-03-08 at 09 36 49

**Comment by Resaki1** [comment]
> I've noticed that the striped background using the accent color of hidden columns is quite hard to see for some colors (e.g. poker purple, lean lilac, planning pink). At first sight, I thought it was a bug that for some columns the background isn't stripped.
> 
> Screenshot 2023-03-08 at 09 34 44
> 
> Also, since you've removed the spacer on the left and right sides, columns are now of different widths. Screenshot 2023-03-08 at 09 36 36
> 
> Screenshot 2023-03-08 at 09 36 49

I tried to fix both issues, please check if it's fine for you

**Comment by bitionaire** [comment]
@brandstetterm plz have a l00k

**Pull request merged**